### DISCUSSION TITLE
Найцікавіші новини місяця: листопад 2024

### DISCUSSION THREAD

**Discussion by rkiyanchuk** [shopokodu/community#75] [Теми для подкасту]
* [Go 15 років](https://go.dev/blog/15years)
* [Рекордний стрім](https://about.netflix.com/en/news/jake-paul-vs-mike-tyson-over-108-million-live-global-viewers) на понад 100 мільйонів глядачів від Netflix
* [S.T.A.L.K.E.R. 2](https://robotdreams.cc/blog/605-stalker-2-is-the-game-worth-15-years-of-waiting)
* https://github.blog/changelog/2024-11-05-notice-of-breaking-changes-for-github-actions/
* [fun] https://kernelnewbies.org/Linux_6.12 (QR code panic screen) 
* [Redis is trying to take over the all the OSS Redis libraries](https://github.com/redis-rs/redis-rs/issues/1419) ([reddit](https://www.reddit.com/r/programming/comments/1h0dv1x/redis_is_trying_to_take_over_the_all_the_oss/))
* [Paid Software on Flathub](https://www.reddit.com/r/linux/comments/1h81a9d/paid_software_is_coming_to_flathub/)
* Тема для @malor: [ReiserFS Has Been Deleted From The Linux Kernel](https://www.phoronix.com/news/ReiserFS-Deleted-Linux-6.13)

**Comment by eugeneradionov** [comment]
* [OpenAI's new model tried to avoid being shut down](https://www.transformernews.ai/p/openais-new-model-tried-to-avoid)

* [Е-гривня у 2025 році — НБУ планує відкритий тест електронної валюти](https://itc.ua/ua/novini/e-gryvnya-u-2025-rotsi-nbu-planuye-vidkrytyj-test-elektronnoyi-valyuty/)
